### PR TITLE
Sanjana nonleaf bddl changes

### PR DESCRIPTION
**Pull request by sanjanasrivastava**

### CODE REVIEW

File: bddl/activity_definitions/cleaning-up-branches-and-twigs/problem0.bddl [hunk 1]
```diff
@@ -1,11 +1,11 @@
-(define (problem cleaning-up-branches-and-twigs_0)
```
Comment (ChengshuLi) [inline bddl/activity_definitions/cleaning-up-branches-and-twigs/problem0.bddl:1]: Let's actually do branch.n.02 in this task. I think stalk is a bit too high-level. My portion of the BDDLs already assume stalk.n.02 is not a leaf.

File: bddl/activity_definitions/cleaning_shoes/problem0.bddl [hunk 2]
```diff
@@ -4,28 +4,28 @@
...
         stain.n.01_1 - stain.n.01
         dust.n.01_1 - dust.n.01
-     	soap.n.01_1 - soap.n.01
+     	bar_soap.n.01_1 - bar_soap.n.01
     	bed.n.01_1 - bed.n.01
     	floor.n.01_1 - floor.n.01
     	rag.n.01_1 - rag.n.01
     	towel.n.01_1 - towel.n.01
-    	shoe.n.01_1 shoe.n.01_2 shoe.n.01_3 shoe.n.01_4 - shoe.n.01
+    	gym_gym_shoe.n.01_1 gym_shoe.n.01_2 gym_shoe.n.01_3 gym_shoe.n.01_4 - gym_shoe.n.01
```
Comment (ChengshuLi) [inline bddl/activity_definitions/cleaning_shoes/problem0.bddl:12]: double gym_gym_shoe. Is your sanity check script catching these?

File: bddl/activity_definitions/cooking-breakfast/problem0.bddl [hunk 3]
```diff
@@ -11,7 +11,7 @@
         countertop.n.01_1 - countertop.n.01
         frying_pan.n.01_1 - frying_pan.n.01
         saucepan.n.01_1 - saucepan.n.01
-        spoon.n.01_1 - spoon.n.01
+        wooden_spoon.n.01_1 - wooden_spoon.n.01
```
Comment (ChengshuLi) [inline bddl/activity_definitions/cooking-breakfast/problem0.bddl:14]: wooden_spoon.n.01 is the wrong synset. It should be wooden_spoon.n.02.

File: bddl/activity_definitions/make-frozen-lemonade/problem0.bddl [hunk 4]
```diff
@@ -7,7 +7,7 @@
         lemon_juice.n.01_1 - lemon_juice.n.01
         pitcher.n.02_1 pitcher.n.02_2 - pitcher.n.02
         lemonade.n.01_1 - lemonade.n.01
-        spoon.n.01_1 - spoon.n.01
+        wooden_spoon.n.01_1 - wooden_spoon.n.01
```
Comment (ChengshuLi) [inline bddl/activity_definitions/make-frozen-lemonade/problem0.bddl:10]: same

File: bddl/activity_definitions/make-lemonade/problem0.bddl [hunk 5]
```diff
@@ -7,7 +7,7 @@
         lemon_juice.n.01_1 - lemon_juice.n.01
         pitcher.n.02_1 - pitcher.n.02
         lemonade.n.01_1 - lemonade.n.01
-        spoon.n.01_1 - spoon.n.01
+        wooden_spoon.n.01_1 - wooden_spoon.n.01
```
Comment (ChengshuLi) [inline bddl/activity_definitions/make-lemonade/problem0.bddl:10]: same

File: bddl/activity_definitions/make-warm-drinks-for-the-holidays/problem0.bddl [hunk 6]
```diff
@@ -6,7 +6,7 @@
         honey.n.01_1 - honey.n.01
         lemon_juice.n.01_1 - lemon_juice.n.01
         water.n.06_1 - water.n.06
-        beverage.n.01_1 - beverage.n.01
+        hot_toddy.n.01_1 - hot_toddy.n.01
```
Comment (ChengshuLi) [inline bddl/activity_definitions/make-warm-drinks-for-the-holidays/problem0.bddl:9]: you might need to update the csv for this task later, just a heads up

File: bddl/activity_definitions/polishing_shoes/problem0.bddl [hunk 7]
```diff
@@ -3,19 +3,19 @@
 
     (:objects
         stain.n.01_1 - stain.n.01
-    	shoe.n.01_1 shoe.n.01_2 - shoe.n.01
```
Comment (ChengshuLi) [inline bddl/activity_definitions/polishing_shoes/problem0.bddl:6]: For some of these tasks, maybe it's okay to leave it be `shoe.n.01`? What do you think?

File: bddl/activity_definitions/praeparing-lunch-box/problem0.bddl [hunk 8]
```diff
@@ -6,7 +6,7 @@
         chopping_board.n.01_1 - chopping_board.n.01
         countertop.n.01_1 - countertop.n.01
         packing_box.n.02_1 - packing_box.n.02
-        sandwich.n.01_1 - sandwich.n.01
```
Comment (ChengshuLi) [inline bddl/activity_definitions/praeparing-lunch-box/problem0.bddl:9]: Same. Maybe it's okay for it to be `sandwich.n.01`?

File: bddl/activity_definitions/put-togethera-basic-pruning-kit/problem0.bddl [hunk 9]
```diff
@@ -4,15 +4,13 @@
     (:objects
      	pruner.n.02_1 - pruner.n.02
     	floor.n.01_1 - floor.n.01
-    	pruning_saw.n.01_1 - pruning_saw.n.01
```
Comment (ChengshuLi) [inline bddl/activity_definitions/put-togethera-basic-pruning-kit/problem0.bddl:7]: Maybe replace this with pruner.n.02?

**Review by ChengshuLi — COMMENTED** [review]
LGTM with some minor comments!

File: bddl/activity_definitions/make-warm-drinks-for-the-holidays/problem0.bddl (hunk 6 above)
Comment (sanjanasrivastava) [reply bddl/activity_definitions/make-warm-drinks-for-the-holidays/problem0.bddl:9]: did it on friday!

File: bddl/activity_definitions/put-togethera-basic-pruning-kit/problem0.bddl (hunk 9 above)
Comment (sanjanasrivastava) [reply bddl/activity_definitions/put-togethera-basic-pruning-kit/problem0.bddl:7]: hm, for some reason I already have pruner.n.02

**Review by ChengshuLi — APPROVED** [review]

**Pull request merged**